webrender/src/resource_cache.rs

```diff
@@ -445,6 +471,23 @@ impl ResourceCache {
         }
     }
 
+    pub fn request_geometry(&mut self,
```

Comment:
It does; the difference is for vector images we need to record the dimensions we ask the svg renderer to rasterize and if a different dimensions is requested the webrender should treat it as a new regular image. Otherwise it should just behave like regular images.